cli/src/main/kotlin/com/meamoria/lexurgy/cli/server/Server.kt

```diff
@@ -39,36 +63,64 @@ data class StringCollector(val strings: MutableList<String> = mutableListOf()) :
     }
 }
 
-fun runServer(changes: Path) {
-    val changer = soundChangerFromLscFile(changes)
+fun applyChanges(changer: SoundChanger?, request: ApplyChangesRequest) {
+    if (changer == null) throw Exception("No sound changer loaded.")
```

Comment:
Don't just throw `Exception`. Throw a custom exception type, allowing subsequent processing of errors if necessary.

Reply:
Not sure why I didn’t do the same thing here that I did earlier candidly; there doesn’t seem to be an exception type that makes sense for this, so I’m going to add one as well